### PR TITLE
Lapa22

### PR DESCRIPTION
**Pull request by zachandfox**

### CODE REVIEW

File: tables/rx_plasma/Lapa22-plasma.csv [hunk 1]
```diff
@@ -0,0 +1,251 @@
+﻿ref_name,rx_name,subject_name,collection_date,cumulative_group
+Lapa22,Sputnik_V_3mo_2,Sputnik_V,2021-12-31,Sputnik_V
```
Comment (KaimingTao) [inline tables/rx_plasma/Lapa22-plasma.csv:2]: The date of all plasmas are the same, please use the corresponding date.

File: tables/subject_history/Lapa22-pth.csv [hunk 2]
```diff
@@ -0,0 +1,20 @@
+﻿ref_name,subject_name,event,event_date_cmp,event_date,location,iso_name,vaccine_name,severity
+Lapa22,BNT162b2,1st dose,~,2021-01-01,Italy+Russia,NULL,BNT162b2,NULL
+Lapa22,BNT162b2,2nd dose,~,2021-02-01,Italy+Russia,NULL,BNT162b2,NULL
+Lapa22,BNT162b2,2nd dose isolation,~,2021-02-15,Italy+Russia,NULL,NULL,NULL
+Lapa22,BNT162b2,2nd dose isolation,~,2021-05-01,Italy+Russia,NULL,NULL,NULL
+Lapa22,BNT162b2,2nd dose isolation,~,2021-08-01,Italy+Russia,NULL,NULL,NULL
+Lapa22,BNT162b2,isolation,~,2021-12-31,Italy+Russia,NULL,NULL,NULL
```
Comment (KaimingTao) [inline tables/subject_history/Lapa22-pth.csv:7]: I think BNT162b2 group didn't get infected. If so, please remove `isolation` event.

File: tables/subjects.csv [hunk 3]
```diff
@@ -1055,6 +1055,10 @@ Kitchin21,Ad26.COV2.S,Human,NULL,13
 Kitchin21,Ad26.COV2.S_BTI,Human,NULL,6
 Kuzmina21,BNT,Human,NULL,10
 Kuzmina21,CP,Human,NULL,10
+Lapa22,BNT162b2,Human,NULL,1
```
Comment (KaimingTao) [inline tables/subjects.csv:1088]: Please collect the number of subjects and correct the last row of each record.

File: tables/rx_plasma/Lapa22-plasma.csv (hunk 1 above)
Comment (KaimingTao) [reply tables/rx_plasma/Lapa22-plasma.csv:2]: Please use all lowercase letter to name files.

File: tables/subjects.csv (hunk 3 above)
Comment (zachandfox) [reply tables/subjects.csv:1088]: I thought this was created automatically

Comment (KaimingTao) [reply tables/subjects.csv:1088]: Err, It's not automatically generated.

Comment (philiptzou) [reply tables/subjects.csv]: The first four columns can be automatically generated but the last one should be updated manually.

**Pull request merged**